onnxscript/rewriter/ort_fusions/skip_normalization.py

```diff
@@ -22,7 +22,15 @@ def __init__(self, name: str, has_bias: bool = False, bias_pre_add: bool = False
     def pattern(self, op, input, skip, gamma, bias, epsilon, stash_type):
         if self._has_bias and self._bias_pre_add:
             input = op.Add(input, bias)
-        skip_sum = op.Add(input, skip)
+
+        # Support different combinations of addition of input and skip
+        skip_sum_pattern_1 = op.Add(skip, input)
```

Comment:
Do you see both these patterns arising in examples? Or, is it just to try it out?

Reply:
The Whisper model consists of this pattern. I've seen this in a couple other models in the benchmark as well.

Reply:
Just to clarify my question: I mean, have you seen *both* variations of doing the Add (both `Add(skip,input)` and `Add(input,skip)`)?